cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -1,149 +1,4 @@
-// TODO: Remove after Capital Framework v4 is merged in.
-
-.a-label {
-    display: inline-block;
-}
-
-.a-label__heading {
-    .heading-4();
-
-    display: block;
-
-    // Overwrites heading-5 margin.
-    margin-bottom: 10px;
-}
-
-
-@input-border__focused: @pacific;
-@input-bg__disabled: @gray-5;
-
-.m-form-field {
-    .a-text-input {
-        box-sizing: border-box;
-        width: 100%;
-    }
-
-    .a-label + .a-text-input {
-        margin-top: unit( 5px / @base-font-size-px, em );
-    }
-
-    &__checkbox,
-    &__radio {
-        .a-label {
-            vertical-align: middle;
-            cursor: pointer;
-
-            &:before {
-                display: inline-block;
-
-                border: 1px solid @input-border;
-                height: unit( 18px / @base-font-size-px, em );
-                width: unit( 18px / @base-font-size-px, em );
-                margin-right: 10px;
-
-                background-color: @input-bg;
-                content: '';
-                vertical-align: top;
-
-                .lt-ie9 & {
-                    display: none !important;
-                }
-            }
-
-            &:hover:before {
-                border-color: @input-border__focused;
-            }
-        }
-
-        .a-checkbox,
-        .a-radio {
-            .u-visually-hidden();
-
-            .lt-ie9 & {
-                height: unit( 20px / @base-font-size-px, em );
-                width: unit( 20px / @base-font-size-px, em );
-                border: 0;
-                float: left;
-                margin: 1em;
-                position: static;
-                width: auto;
-                clear: both;
-            }
-
-            &:disabled + .a-label {
-                cursor: not-allowed;
-
-                &:before {
-                    border-color: @input-border;
-                    background: @input-bg__disabled;
-                }
-            }
-        }
-    }
-
-    &__checkbox {
-        .a-label {
-            &:before {
-                font-family: 'CFPB Minicons';
-                text-align: center;
-                line-height: unit( 18px / @base-font-size-px, em );
-            }
-        }
-
-        .a-checkbox {
-            &:focus + .a-label:before,
-            &:hover + .a-label:before {
-                border-color: @input-border__focused;
-                outline: 1px solid @input-border__focused;
-            }
-
-            &:checked + .a-label:before {
-                content: @cf-icon-approved;
-            }
-        }
-    }
-}
-
-.o-form {
-    &_group {
-        margin-bottom: unit( 30px / @base-font-size-px, em );
-    }
-
-    &_fieldset {
-        // Overwrite Normalize.
-        border: none;
-        margin: 0;
-        padding: 0;
-
-        .m-field + .m-field {
-            margin-top: unit( 10px / @base-font-size-px, em );
-        }
-    }
-}
-
-.o-form_fieldset .a-text-input {
```

Comment:
`o-form_fieldset` is referenced in the CF docs but doesn't have this style, so the consequence looks like any text inputs won't be 100% when inside a fieldset, if that's in use somewhere.